shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -11,12 +11,13 @@
 
 public final class MetricsNames {
     // Metrics in Segment Store Service
-    // host stats
+    // Segment-related stats
     public static final String SEGMENT_CREATE_LATENCY = "segment_create_latency_ms"; // Timer
     public static final String SEGMENT_READ_LATENCY = "segment_read_latency_ms";     // Timer
     public static final String SEGMENT_WRITE_LATENCY = "segment_write_latency_ms";   // Timer
     public static final String SEGMENT_READ_BYTES = "segmentstore.segment_read_bytes";            // Dynamic Counter
     public static final String SEGMENT_WRITE_BYTES = "segmentstore.segment_write_bytes";          // Dynamic Counter
+    public static final String SEGMENT_WRITE_EVENTS = "segment_write_events";                     // Dynamic Counter
```

Comment:
To be consistent with current naming, it should be "segmentstore.segment_write_events". Dynamic metrics have group prefix in MetricsNames.java